tasks/sealights/sealights-go-instrumentation/0.1/sealights-go-instrumentation.yaml

```diff
@@ -0,0 +1,151 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sealights-go-instrumentation
+  labels:
+    konflux-ci/sealights: "true"
+    konflux-ci/sealights-language: "go"
+spec:
+  description: >-
+    This Tekton task automates the process of instrumenting Go code with Sealights for quality analytics and testing.
+    It retrieves the source code from a trusted artifact, instruments the code with Sealights, and then creates
+    a new trusted artifact with the instrumented code. The task can be triggered by either Pull Request or other events.
+  results:
+    - name: sealights-bsid
+      type: string
+      description: "A unique identifier generated for the current sealights build session."
+    - name: sealights-build-name
+      type: string
+      description: "A unique build name generated using the commit SHA and current date to prevent conflicts during test reruns."
+    - name: source-artifact
+      description: The Trusted Artifact URI pointing to the artifact with
+        the application source code.
+      type: string
+  params:
+    - name: source-artifact
+      description: The Trusted Artifact URI pointing to the artifact with
+        the application source code.
+      type: string
+    - name: go-version
+      type: string
+      description: "The Go version to use with the 'ubi8/go-toolset' image, in the format '1.x.y' (e.g., '1.21.3')."
+    - name: sealights-secret
+      type: string
+      description: "The name of the Openshift secret containing Sealights credentials."
```

Comment:
maybe mention that the key `token` is expected and point to the docs which describes how to get it?

Reply:
cool. Let me find the docs.